### PR TITLE
MGMT-16883: Update external jobs to use ocm-2.11 image-sets

### PR DESCRIPTION
**Pull request by adriengentil**
Update assisted to use ocm-2.11 for jobs on master, 4.16 and 4.17
branches.

### CODE REVIEW

File: ci-operator/config/openshift/installer/openshift-installer-release-4.17.yaml [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
     namespace: edge-infrastructure
     tag: assisted-installer-controller
   assisted-service:
-    name: ocm-2.10
+    name: ocm-2.11
     namespace: edge-infrastructure
     tag: assisted-service
   assisted-test-infra:
-    name: ocm-2.10
+    name: ""
```
Comment (gamli75) [inline ci-operator/config/openshift/installer/openshift-installer-release-4.17.yaml:23]: Is there a reason why you removed it?

Comment (adriengentil) [reply ci-operator/config/openshift/installer/openshift-installer-release-4.17.yaml:23]: nope, good catch, thanks!

**Review by gamli75 — APPROVED** [review]

**Pull request merged**